actionsystemtest/lib/action_system_test/driver_adapters.rb

```diff
@@ -0,0 +1,46 @@
+module ActionSystemTest
+  # == Action System Testing Driver Adapters
+  #
+  # By default Rails supports Capybara with the Selenium Driver. Rails provides
+  # configuration setup for using the selenium driver with Capybara.
+  # Additionally Rails can be used as a layer between Capybara and its other
+  # supported drivers: +:rack_test+, +:selenium+, +:webkit+, or +:poltergeist+.
+  #
+  # *{RackTest}[https://github.com/jnicklas/capybara#racktest]
+  # *{Selenium}[http://seleniumhq.org/docs/01_introducing_selenium.html#selenium-2-aka-selenium-webdriver]
+  # *{Webkit}[https://github.com/thoughtbot/capybara-webkit]
+  # *{Poltergeist}[https://github.com/teampoltergeist/poltergeist]
+  #
+  # === Driver Features
+  #
+  # |                 | Default Browser       | Supports Screenshots? |
+  # | --------------- | --------------------- | --------------------- |
+  # | Rails' Selenium | Chrome                | Yes                   |
+  # | Rack Test       | No JS Support         | No                    |
+  # | Selenium        | Firefox               | Yes                   |
+  # | WebKit          | Headless w/ Qt        | Yes                   |
+  # | Poltergeist     | Headless w/ PhantomJS | Yes                   |
```

Comment:
Can we just pick WebKit or Poltergeist? If they offer the same browser compatibility, we don't need both.